buildSrc/src/main/kotlin/otel.java-conventions.gradle.kts

```diff
@@ -250,3 +250,15 @@ afterEvaluate {
     }
   }
 }
+
+codenarc {
```

Comment:
These seemed tiny enough to not extract into files. But happy to do so anyways if anyone prefers it.